src/api/index.ts

```diff
@@ -153,6 +154,12 @@ router.get(
 		const githubInstallationId = Number(req.params.installationId);
 
 		try {
+			if (!req.session.jiraHost) {
+				req.log.warn({ req, res }, "Missing Jira Host");
+				res.status(500).send("Missing Jira Host");
+				return;
```

Comment:
Do we need to add a check here? Why do we need Jira host for our admin endpoints at all?

(If we really need it the response status should be 400)

Reply:
Because it's being used to get the subscription below and the typings were erroring.  Now that I look at it though, seems like the logic of this API itself is wrong and the jiraHost should be passed in the API POST body and not part of the session.